core-services/prow/02_config/_config.yaml

```diff
@@ -354,6 +355,7 @@ plank:
         mediaTypes:
           log: text/plain
       gcs_credentials_secret: gce-sa-credentials-gcs-qe-private-deck
+      grace_period: 2h0m0s
```

Comment:
can we change the global value?

Reply:
Change global value will impact all jobs, while change here will limit the impact to our jobs in openshift-tests-private and verification-tests.